src/class-dashboard-links.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Parse.ly Dashboard Links utility class.
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+use WP_Post;
+
+/**
+ * Utility methods to build and generate dashboard links.
+ *
+ * @since 3.2.0
+ */
+class Dashboard_Links {
```

Comment:
Right now, the methods are only dealing with the creation/handling of single links at a time (rather than a collection of dashboard links), so I think class name (and file) should be referenced in the singular (`Dashboard_Link`).